src/make-http-request.coffee

```diff
@@ -1,4 +1,5 @@
-request = require 'superagent'
+cacheModuleConfig = { defaultExpiration: 10, verbose: true }
```

Comment:
I'll turn verbose off before mege but it's useful for testing. Also look to extract the expiration config